hilgendorf_moveit_config/config/joint_limits.yaml

```diff
@@ -0,0 +1,61 @@
+joint_limits:
+  right_wrist_2_joint:
+    has_velocity_limits: true
+    max_velocity: 3.2
+    has_acceleration_limits: false
+    max_acceleration: 0
```

Comment:
The `max_acceleration` is set to zero.  Was this done on purpose, instead of setting it to `5` like I suggested above.